### DISCUSSION TITLE
dailydev should be aware of holidays also, not only weekends.

### DISCUSSION THREAD

**Discussion by pablominofortinet** [dailydotdev/daily#1680] [Feature Requests]
I lost my streak to Christmas holidays....

On your profile you should be able to mark holidays that you have, either from country or from religion. It should have preloaded holidays from all religions and from all the countries, 
There are nomads and people just like me that live in more than one country, so you should be able to pick more than one country holidays. I live traveling between Uruguay and Paraguay and apart from holidays from these 2 countries I have holidays from the USA which is my main employer. 
There are people that work for Jewish company but are catholic, hence they have both holidays, 

You should be able to pick fixed dates, like every 25th December for holidays
You should be able to pick second Monday of March for holidays
You should be able to pic all the holidays preloaded by a certain country or religion.

Also, you should be able to pick other weekends other than friday and saturday, and saturday and sunday... people working at health have moving shifts, even in IT or software development

**Comment by idoshamun** [comment]
Hi @pablominofortinet,

Thanks for the feedback. Since supporting holidays has tons of edge cases, we decided that our streak system will have a restore feature, giving users the flexibility to do it whenever they'd like while paying with reputation points.
